src/state/slices/opportunitiesSlice/constants.ts

```diff
@@ -118,13 +118,13 @@ export const DEFI_PROVIDER_TO_METADATA: Record<DefiProvider, DefiProviderMetadat
   },
   [DefiProvider.CosmosSdk]: {
     provider: DefiProvider.CosmosSdk,
-    icon: 'https://assets.coincap.io/assets/icons/256/atom.png',
+    icon: 'https://coin-images.coingecko.com/coins/images/1481/large/cosmos_hub.png?1696502525',
     color: '#C5B5F2',
     url: 'https://app.shapeshift.com',
   },
   [DefiProvider.ThorchainSavers]: {
     provider: DefiProvider.ThorchainSavers,
-    icon: 'https://assets.coincap.io/assets/icons/rune@2x.png',
+    icon: 'https://coin-images.coingecko.com/coins/images/6595/large/Rune200x200.png?1696506946',
```

Comment:
blocking: same issue here

Reply:
Updated in [5fa1cb5](https://github.com/shapeshift/web/pull/7298/commits/5fa1cb537de3788088524efdd98220955cb62029)